src/status_im2/contexts/quo_preview/standart_auth/keycard_pin_entry.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im2.contexts.quo-preview.standart-auth.keycard-pin-entry
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [utils.re-frame :as rf]
+            [status-im2.common.standard-auth.keycard.view :as standard-auth]))
+
+(def descriptor
+  [{:key     :theme
+    :type    :select
+    :options [{:key :dark}
+              {:key :light}
+              {:key nil :value "System"}]}])
+
+(defn preview
+  []
+  (let [state (reagent/atom {})]
+    (fn []
+      [preview/preview-container {:state state :descriptor descriptor}
+       [quo/button
+        {:container-style {:margin-horizontal 40}
+         :on-press        #(rf/dispatch [:show-bottom-sheet
+                                         {:content standard-auth/enter-keycard-pin-sheet}])}
+        "See in bottom sheet"]
+       [rn/view {:padding-vertical 60}
```

Comment:
I like the approach you used here, you are previewing the component in and out of the bottom sheet because this shows the component works independently of any particular parent component. It would be good if you did the same in the other preview namespaces.